modules/Link.js

```diff
@@ -28,15 +29,19 @@ class Link extends React.Component {
     activeClassName: '',
     style: {},
     activeStyle: {},
-    isActive: (location, to, props) => {
-      return pathIsActive(
-        to.pathname,
-        location.pathname,
-        props.activeOnlyWhenExact
-      ) && queryIsActive(
-        to.query,
-        location.query
-      )
+    isActive: (location, to, props, router) => {
+      if (router) {
```

Comment:
I wouldnt check for router here. Move it up to wrap the construction of all props for the eventual a tag. I.e., in the case of no router context, this renders a normal a tag.